assets/javascripts/discourse/routes/admin-plugins-explorer.js.es6

```diff
@@ -4,19 +4,36 @@ export default Discourse.Route.extend({
   controllerName: "admin-plugins-explorer",
 
   model() {
-    const p1 = this.store.findAll("query");
```

Comment:
probably worth renaming this to queryPromise and so on ... p1,p2,p3 is somewhat confusing.